yjit/src/backend/ir.rs

```diff
@@ -1229,6 +1229,10 @@ impl AssemblerDrainingIterator {
         self.index += 1;
         self.insns.next().map(|insn| (index, insn))
     }
+
+    pub fn peek(&mut self) -> Option<&Insn> {
```

Comment:
Can you add a comment explaining what this method is used for?

Reply:
addressed in https://github.com/ruby/ruby/pull/7453/commits/a6949591c1fe524a1b0331958cd5a6bcc8187517